src/plugins/cordova-plugin-camera/sim-host.js

```diff
@@ -1,6 +1,12 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 
 var dialog = require('dialog');
+var telemetry = require('../../client-common/client-telemetry-helper');
+
+var baseProps = {
+    plugin: 'cordova-plugin-camera',
```

Comment:
I initially thought of doing something like that, but all plugin code gets bundled into sim-host.js under the simulation folder in the app, so we can't use things like `__dirname`. Essentially, we want to programmatically get the "design time" folder name, when the "run time" is different. I don't see a way of doing this without hardcoding something (if we don'T hardcode the plugin name, we'll need to hardcode the path to its folder, so we don't really gain anything).